client/post-editor/post-editor.jsx

```diff
@@ -267,7 +266,7 @@ export class PostEditor extends React.Component {
 			<div className={ classes }>
 				<PageViewTracker path={ this.props.analyticsPath } title={ this.props.analyticsTitle } />
 				<QueryPreferences />
-				{ isEnabled( 'gutenberg/opt-in' ) && <EditorGutenbergOptInNotice /> }
+				<EditorGutenbergOptInNotice />
```

Comment:
Not a blocker, but I'm curious about your thinking in moving the render logic into the components themselves. It felt more clear to me to have the conditional at a higher level (i.e., when looking at `post-editor.jsx`, you can tell that things will not always be displayed, without having to dig into the component level).

Granted, given that I had to import `isEnabled` into this file the first time I'm assuming other components are doing the same thing already.